Assignment done

arron-tij opened the pull request:
I have tried completing all the tasks including login/register with bootstrap.  
Thankyou for this assignment I got to learn a lot. :)

Eerie16 commented:
- The BookCopy model represents each physical copy of a particular book available in the library. So, if book A has 5 copies available, there will be 5 BookCopy instances corresponding to the book. So, modify your code according to it. Use the status and borrower fields for lending and returning. 
- For rating, you need to store the rating given by each user so that he can see what he gave and also modify it.

Great work on the authentication app, try to also implement login view. Make the necessary commits and drop a comment when you're done.

arron-tij commented:
Does this mean I can order more than one copy of the same book then?

Eerie16 commented:
Yes, a person can order any instance of the bookcopy model whose status is available. So, more than one instance of the same book is allowed

arron-tij commented:
- Changed the loan and return function to operate on the 'status' field.
 - Near the rate button , the user is displayed his past rating and when he rates again the rating is 
   modified accordingly such that his last rating only counts

Eerie16 commented:
Good work on the rating section but your implementation has a serious bug. If more than 1 copies of the book are available, all are issued to the same person. Good overall work.
